test/e2e/lib/gutenberg/blocks/contact-info-block-component.js

```diff
@@ -12,6 +12,48 @@ class ContactInfoBlockComponent extends GutenbergBlockComponent {
 	static blockTitle = 'Contact Info';
 	static blockName = 'jetpack/contact-info';
 	static blockFrontendSelector = By.css( '.entry-content .wp-block-jetpack-contact-info' );
+
+	async fillUp( {
+		email,
+		phoneNumber,
+		streetAddress,
+		addressLine2,
+		addressLine3,
+		city,
+		state,
+		zipCode,
+		country,
+		linkToGmaps,
+	} ) {
+		const blockQuery = `div[id='${ this.blockID.slice( 1 ) }']`;
```

Comment:
This can be simplified down to a selector (locator):
```js
const blockSelector = By.id( this.blockID );
```
...and then if the string value is needed it can be requested via `blockSelector.value` (will return i.e. `"#block-12345"`).

Reply:
This actually made me realize that [here](https://github.com/Automattic/wp-calypso/blob/bedb530aa05d2da5b063f31a9112e4ed8c958b13/test/e2e/lib/gutenberg/blocks/layout-grid-block-component.js#L30-L33) we're setting a wrong format of the `blockID`. By default, it's set with the hash symbol, but when acquired via `getAttribute( 'id' )` - it's not there which can cause confusion at some point. I will update in this PR as it's a small fix.

Reply:
Updated in https://github.com/Automattic/wp-calypso/pull/45796/commits/d7114aeff40abe9a4a58428f07a72cbae2611cad

Reply:
> `div[id='${ this.blockID.slice( 1 ) }']`;

Ah true, I don't know why I used the attribute selector here.

> This actually made me realize that here we're setting a wrong format of the blockID. By default, it's set with the hash symbol (#block-123), but when acquired via getAttribute( 'id' ) - it's not there (block-123), which can cause confusion at some point. I will update in this PR as it's a small fix.

👍🏻 Thanks for catching that.